### PR TITLE
feat: Trigger SMS

### PR DESCRIPTION
**Pull request by ALFAROSO**
[Jira-BE - Harvest Log - Create Harvest Log - Trigger SMS](https://team-voyagers.atlassian.net/jira/software/projects/P2/boards/2?assignee=712020%3Aa4d78f0d-920a-427a-a5ce-b36979797950&selectedIssue=P2-302)

[Jira-BE - Payroll - Run Payroll - Trigger SMS](https://team-voyagers.atlassian.net/jira/software/projects/P2/boards/2?assignee=712020%3Aa4d78f0d-920a-427a-a5ce-b36979797950&selectedIssue=P2-303)

twilio

### CODE REVIEW

**Comment by ALFAROSO** [comment]
https://github.com/LangaraVoyagers/project-2-api/assets/87240930/39f67817-b5c1-4819-a95d-9ddc4b3a966c

File: services/messageService.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  pickerName: string,
+  netAmount: number,
+  unit: string,
+  product: string,
+  payment: number,
+  currency: string
+) => {
+  try {
+    const message = await client.messages.create({
+      body: `Hi ${pickerName}, you have collected ${netAmount} ${unit.toLowerCase()} of ${product.toLowerCase()}. That means you've earned ${payment} ${currency} for your next paycheck.`,
```
Comment (ReinhardtBotha) [inline services/messageService.ts:23]: I think it should be currency and then amount.

File: controllers/harvest-log.controllers.ts [hunk 2]
```diff
@@ -115,6 +120,42 @@ const create = async (req: Request, res: Response) => {
...
+      let unitUser = (savedHarvestLog.season as any).unit.name;
+      unit = pluralize(unitUser);
+    } else {
+      unit = (savedHarvestLog.season as any).unit.name;
+    }
+
+    let currency = (savedHarvestLog.season as any).currency.name;
+
+    let message = `Hi ${pickerName}, you have collected ${netAmount} ${unit.toLowerCase()} of ${product.toLowerCase()}. That means you've earned ${payment} ${currency} for your next paycheck.`;
+    console.log(message);
```
Comment (ReinhardtBotha) [inline controllers/harvest-log.controllers.ts:146]: remove console log

File: .env.demo [hunk 3]
```diff
@@ -2,3 +2,5 @@ PORT="PORT WHERE SERVER IS RUNNING"
 MONGO_CONNECTION_STRING="mongodb+srv://<username>:<password>@cluster0.jsllulh.mongodb.net/<database>?retryWrites=true&w=majority"
 JWT_PRIVATE_KEY="JWT PRIVATE KEY FOR ENCRYPTION"
 JWT_EXP_DURATION="EXPIRE DURATION FOR JWT KEY ex 1800s, 1d"
+TWILIO_ACCOUNT_SID=AC0bb22eb3fd211743b07543c12c2f500e
```
Comment (ReinhardtBotha) [inline .env.demo:5]: are these the actual keys or just examples?

**Review by ReinhardtBotha — APPROVED** [review]

File: .env.demo (hunk 3 above)
Comment (ALFAROSO) [reply .env.demo:5]: The actual keys

**Review by whamester — APPROVED** [review]

**Pull request merged**